src/db.rs

```diff
@@ -110,6 +122,22 @@ impl Db {
         }
     }
 
+    #[cfg(feature = "ck3")] // vic3 happens not to use
+    pub fn cf_has_property(
```

Comment:
should be lc_*

Reply:
I was searching for `_cf` and missed it :)